packages/astro/src/@types/astro.ts

```diff
@@ -2164,6 +2164,35 @@ export interface AstroUserConfig {
 			 * ```
 			 */
 			schema?: EnvSchema;
+
+			/**
+			 * @docs
+			 * @name experimental.env.validateSecretsOnStart
+			 * @kind h4
+			 * @type {boolean}
+			 * @default `false`
+			 * @version 4.11.3
```

Comment:
I need to update it once we're confident it can be merged (not the case yet)

Reply:
It's as good as it can be until I see it live! This looks right to me, though.